pytest_fixtures/component/taxonomy.py

```diff
@@ -247,6 +256,15 @@ def function_sca_manifest():
         yield manifest
 
 
+@pytest.fixture
+def function_golden_ticket_sca_manifest():
```

Comment:
What is the reason for adding another function-scoped SCA manifest fixture? This is a duplicate of `function_sca_manifest`.

Reply:
Looking further down, I'm assuming this is for cases when two manifests are needed in one test. If this is accurate, then can you please change the name of this fixture to indicate its purpose more clearly?